### PR TITLE
[T3A2][W13-A2]Song Zhiwen

### PR DESCRIPTION
**Pull request by zzzzwen**

### CODE REVIEW

File: src/seedu/addressbook/commands/CommandResult.java [hunk 1]
```diff
@@ -11,7 +11,7 @@
 public class CommandResult {
 
     /** The feedback message to be shown to the user. Contains a description of the execution result */
-    public final String feedbackToUser;
+    protected final String feedbackToUser;
```
Comment (okkhoy) [inline src/seedu/addressbook/commands/CommandResult.java]: why protected? why not private?

**Comment by okkhoy** [comment]
@zzzzwen 
Some comments added. Please ack & close the PR after reading comments.

File: src/seedu/addressbook/commands/CommandResult.java (hunk 1 above)
Comment (zzzzwen) [inline src/seedu/addressbook/commands/CommandResult.java]: should be private since there is no inheritance in this case.

**Comment by zzzzwen** [comment]
noted

**Pull request closed without merging**